### PR TITLE
Unleash the Kraken

### PR DESCRIPTION
**Pull request by Alex-Yates**
# About

This is a post about...

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

File: blog/2020-09/kraken-unleashed/index.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+---
+title: "The Kraken has been unleashed: How to protect your data…"
```
[the remark this answers is not in the document]
Comment (robpearson) [reply blog/2020-09/kraken-unleashed/index.md:2]: I think we should rename the folder/url to match. I'd suggest: `data-guardian-protect-your-data`

File: blog/2020-09/kraken-unleashed/index.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+As the data guardian, you understand more than anyone why deployments fail and how your data can be compromised. Your team needs your expertise to ensure those mistakes are avoided, not just within a single deployment – but within every deployment.
+
+# Step 1: Understand how Octopus Deploy works
+
+In order to contribute to the safe implementation of Octopus Deploy, it’s important to first understand how it works – with a special focus on security.
+
+Octopus Deploy gets its name because its architecture resembles that of an octopus, with a “Server” at the centre, which sends packages and deployment scripts to various targets where files are deployed, and commands are executed.
+
+Octopus users can interact with the server either through a web portal or with scripts, typically using PowerShell, a [REST API](https://octopus.com/blog/change-advisory-boards-dont-work) or a command line utility called [octo.exe](https://octopus.com/docs/octopus-rest-api/octopus-cli).
```
Comment (robpearson) [inline blog/2020-09/kraken-unleashed/index.md:43]:
```suggestion
Octopus users can interact with the server either through a web portal or with scripts, typically using PowerShell, a [REST API](https://octopus.com/blog/change-advisory-boards-dont-work) or a [command line utility](https://octopus.com/docs/octopus-rest-api/octopus-cli).
```

File: blog/2020-09/kraken-unleashed/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+There are several tools and techniques that can be used to perform the deployment. Back in March [I reviewed the most popular options among Octopus Deploy users for deploying SQL Server databases](https://octopus.com/blog/sql-server-deployment-options-for-octopus-deploy). For SQL Server there are a selection of Microsoft, third-party and open source options that can generally be categorised as “migrations-based” or “model-based” solutions.
+
+Whichever tool you use, it’s likely that you’ll either want to pre-install your database tooling on the jump-box or that you’ll want to automate the install/update as part of the deployment process. Pre-installing the tools might be easier in the short term and can speed up your deployments, but it also comes with an administrative overhead and makes your deployment process less portable. Hence, I generally advocate for installing/updating whatever database deployment tools you are using as part of the deployment or on a regular schedule using an [Octopus Runbook](https://octopus.com/docs/runbooks). (Note, this may well require some level of internet access on the jump-box.)
+
+:::hint
```
Comment (robpearson) [inline blog/2020-09/kraken-unleashed/index.md:105]: I added this blurb that we drafted from an older post to give a bit more context on what workers are. @leecarlon Feel free to remove if you don't think it fits. :)

**Review by robpearson — APPROVED** [review]
Great post. My main feedback is around the title and URL to make it clearer as to the value of the post. :)

File: blog/2020-09/kraken-unleashed/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (wordlee) [reply blog/2020-09/kraken-unleashed/index.md:2]: `The Kraken has been unleashed` did make me think of GitKraken which uses similar language.

[the remark this answers is not in the document]
Comment (wordlee) [reply blog/2020-09/kraken-unleashed/index.md:2]: If we lost Kraken from the title we'd need to change `kraken` throughout the text to Octopus as well, I think.

File: blog/2020-09/kraken-unleashed/index.md (hunk 3 above)
Comment (wordlee) [reply blog/2020-09/kraken-unleashed/index.md:105]: Thanks @robpearson. I did remove it as Alex links to the worker docs, so anybody who isn't familiar with them can read the details in the docs.

**Comment by Alex-Yates** [comment]
Ah, if I knew you didn't like the word kraken, I would have completely
rewritten the entire opening.

It's entirely based on a nautical theme about a big and scary octopus (a
kraken), and shifting the perception of it being a scary monster to being a
faithful servant.

**Comment by wordlee** [comment]
We could probably combine the two (Rob's suggestions and Alex's original) into something like:

Taming the Kraken: How to protect your data 

> Ah, if I knew you didn't like the word kraken, I would have completely rewritten the entire opening. It's entirely based on a nautical theme about a big and scary octopus (a kraken), and shifting the perception of it being a scary monster to being a faithful servant.
> […](#)

File: blog/2020-09/kraken-unleashed/index.md (hunk 3 above)
Comment (robpearson) [reply blog/2020-09/kraken-unleashed/index.md:105]: Good call! :)

**Comment by robpearson** [comment]
I like the balance that Lee suggested. :) I like the Kraken character/story and your intro but my point was purely based on clarity for potential readers so they understand what the post is about. :)

**Comment by wordlee** [comment]
Hi Alex, a couple of notes on the edits. I removed the parentheses around some of the phrases, I see why you included them, but the phrases that you'd bracketed occurred as separate sentences after the sentence they were asides to, and so don't need the brackets.
You had a couple of hences in the text. That feels quite archaic (or academic) to me, similar to therefore or hitherto; that might be a personal preference thing though :) 
I love the line about creating the lazy path.

**Comment by Alex-Yates** [comment]
I like "taming the kraken" etc 👍

File: blog/2020-09/kraken-unleashed/index.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
+---
+title: "Tame the Kraken: How to protect your data"
```
Comment (wordlee) [inline blog/2020-09/kraken-unleashed/index.md:2]:
```suggestion
title: "Taming the Kraken: How to protect your data"
```

**Review by wordlee — APPROVED** [review]

**Pull request merged**